src/components/data/reports/reportSettings.js

```diff
@@ -0,0 +1,71 @@
+import moment from 'moment';
+
+const settings = {
+  'overdue': {
+    'queryString': () => {
+      const overDueDate = moment().tz('UTC').format();
+      return `(status.name=="Open" and dueDate < "${overDueDate}") sortby metadata.updatedDate desc`;
+    },
+    'columns' : [
+      'borrower.name',
+      'borrower.barcode',
+      'borrowerId',
+      'dueDate',
+      'loanDate',
+      'loanPolicy.name',
+      'loanPolicyId',
+      'loanId',
+      'feeFine',
+      'item.title',
+      'item.materialType.name',
+      'item.status.name',
+      'item.barcode',
+      'item.callNumberComponents.prefix',
+      'item.callNumberComponents.callNumber',
+      'item.callNumberComponents.suffix',
+      'item.volume',
+      'item.enumeration',
+      'item.chronology',
+      'item.copyNumber',
+      'item.contributors',
+      'item.location.name',
+      'item.instanceId',
+      'item.holdingsRecordId',
+      'itemId']
+  },
+
+  'claimedReturned': {
+    'queryString': () => {
+      return '(status.name=="Open" and action="claimedReturned") sortby metadata.updatedDate desc';
+    },
+    'columns': [
+      'borrower.name',
+      'borrower.barcode',
+      'borrowerId',
+      'dueDate',
+      'loanDate',
+      'loanPolicy.name',
+      'loanPolicyId',
+      'loanId',
+      'feeFine',
+      'item.title',
+      'item.materialType.name',
+      'item.status.name',
+      'item.barcode',
+      'item.callNumberComponents.prefix',
+      'item.callNumberComponents.callNumber',
+      'item.callNumberComponents.suffix',
+      'item.volume',
+      'item.enumeration',
+      'item.chronology',
+      'item.copyNumber',
+      'item.contributors',
+      'item.location.name',
+      'item.instanceId',
+      'item.holdingsRecordId',
+      'itemId',
+    ],
+  }
+};
+
+export default settings;
```

Comment:
Sorry for the late comment. I noticed that the content of the `columns` arrays in both `overdue` and `claimedReturned` objects are the same. If so, wouldn't it be better to move their content into a variable somewhere in constants to avoid duplication?

And also, seems that it is possible not to use quotes in object keys.

Reply:
I address the duplication in the original PR comment.  I can remove the quotes.

Reply:
I'm just trying to be guided in these cases by the principle `YAGNI`. ))